docs/source/user_guide_rag.rst

```diff
@@ -616,7 +616,7 @@ LLMs can be different.
     driver = GraphDatabase.driver(URI, auth=AUTH)
 
     # Create LLM object
-    llm = OpenAILLM(model_name="gpt-3.5-turbo-instruct")
+    llm = OpenAILLM(model_name="gpt-3.5-turbo-0125")
```

Comment:
If it's "just" user guide and example, I'd use a more generic name like `gpt-3.5-turbo` or even `gpt-4o`, so that we do not have to change that too often.

Reply:
https://github.com/neo4j/neo4j-genai-python/pull/125/commits/0688708ecc0e110f3921d587a39220ef4a71e38b